lib/repo_tools.pm

```diff
@@ -253,6 +350,14 @@ sub disable_source_repo {
     }
 }
 
+
+=head2 generate_version
+
+ generate_version();
+
+Generate SLE or openSUSE versions
```

Comment:
Please add some info what `$separator` is about and that it will default to `_` if omitted.